posthog/models/filters/mixins/funnel.py

```diff
@@ -71,6 +75,46 @@ def microseconds_from_days(days):
         return microseconds * FunnelWindowDaysMixin.milliseconds_from_days(days)
 
 
+class FunnelWindowMixin(BaseParamMixin):
+    @cached_property
+    def funnel_window_interval(self) -> Optional[int]:
+        _amt = int(self._data.get(FUNNEL_WINDOW_INTERVAL, "0"))
+        if _amt == 0:
+            return None
+        return _amt
+
+    @cached_property
+    def funnel_window_interval_unit(self) -> Optional[str]:
+        _unit = self._data.get(FUNNEL_WINDOW_INTERVAL_UNIT, None)
```

Comment:
Sorry, realised this a bit later:

Thoughts on storing the unit with `.lower()` here, so we can use it cleanly in `unit_ch()` as well as for the relativedelta infunnel_trends ?